service/service.go

```diff
@@ -86,7 +95,32 @@ func (svc *Service) Close(ctx context.Context) {
 	log.Event(ctx, "graceful shutdown complete", log.INFO)
 }
 
-func registerCheckers(ctx context.Context, hc *healthcheck.HealthCheck) (err error) {
-	// TODO ADD HEALTH CHECKS HERE
-	return
+func registerCheckers(ctx context.Context, hc *healthcheck.HealthCheck, zebedeeClient *zebedee.Client) (err error) {
+	hasErrors := false
+
+	if err = hc.AddCheck("Zebedee", zebedeeClient.Checker); err != nil {
+		hasErrors = true
+		log.Event(ctx, "error adding check for zebedeee", log.ERROR, log.Error(err))
+	}
+
+	if hasErrors {
+		return errors.New("Error(s) registering checkers for healthcheck")
+	}
+	return nil
+}
+
+func getAuthorisationHandlers(cfg *config.Config) api.AuthHandler {
+	auth.LoggerNamespace("dp-sessions-api-auth")
```

Comment:
I think this is generally the same as the app name.

Reply:
I copied this from the observation api - https://github.com/ONSdigital/dp-observation-api/blob/develop/service/service.go, it's also like this in the example in - https://github.com/ONSdigital/dp-authorisation/blob/master/example/main.go